pkg/kube/wait.go

```diff
@@ -46,6 +48,9 @@ type waiter struct {
 // until all are ready or a timeout is reached
 func (w *waiter) waitForResources(created ResourceList) error {
 	w.log("beginning wait for %d resources with timeout of %v", len(created), w.timeout)
+	if err := apiextv1beta1.AddToScheme(scheme.Scheme); err != nil {
```

Comment:
Adding CRDs to the globally shared Scheme (Yay, Singletons!) is probably best done elsewhere. Any idea where?